Formula/llvm.rb

```diff
@@ -225,19 +282,19 @@ def install
   end
 
   def caveats
-    s = <<-EOS.undent
+    caveat_message = <<-EOS.undent
```

Comment:
> This is a bad programming practice

I'm sure you can find articles online that label almost anything bad programming practice. 👍 on Martin's comment.